? cdbakeoven/configure.in.in
? cdbakeoven/cdboshared/CdboCDRom.loT
? cdbakeoven/cdbowidgets/CdboTracksEditorBase.cpp
? cdbakeoven/cdbowidgets/CdboTracksEditorBase.h
? kfortune/prefsui.cpp
? kfortune/prefsui.h
? vimpart/po/Makefile
? vimpart/po/Makefile.in
Index: k3b/configure.in.in
===================================================================
RCS file: /home/kde/kdeextragear-1/k3b/configure.in.in,v
retrieving revision 1.26
diff -u -u -b -r1.26 configure.in.in
--- k3b/configure.in.in	11 Jan 2003 21:07:07 -0000	1.26
+++ k3b/configure.in.in	31 Jan 2003 06:02:23 -0000
@@ -18,7 +18,7 @@
 		OPTIMIZED_ARCH="YES"
 		#EXTRA_CPU_CFLAGS=$(if $CC -march=i686 -S -o /dev/null -xc /dev/null > /dev/null; then echo "-march=i686"; fi)
 	;;
-	ppc)
+	ppc|"Power Macintosh")
 		AC_DEFINE(ARCH_PPC, 1, [WE ARE BUILDING FOR A POWERPC])
 		OPTIMIZED_ARCH="YES"
 	;;
@@ -58,7 +58,7 @@
 	MPG123_PLAT_LIB=libmp686mmx.la
 fi
 # ARCH_PPC
-if test "$ARCH_TYPE" = "ppc"; then
+if test "$ARCH_TYPE" = "ppc" || test "$ARCH_TYPE" = "Power Macintosh"; then
 	ARCH_CFLAGS="$ARCH_CFLAGS -mcpu=ppc"
 	MPG123_PLAT_LIB=libmpppc.la
 fi
@@ -335,7 +335,7 @@
 	arm)               FPM="ARM"     ;;
 	mips)              FPM="MIPS"    ;;
 	sparc)             FPM="SPARC"   ;;
-	ppc|powerpc)       FPM="PPC"     ;;
+	ppc|powerpc|"Power Macintosh") FPM="PPC"     ;;
 	64bit)             FPM="64BIT"   ;;
 	float)             FPM="FLOAT"   ;;
 	*)
Index: cdbakeoven/configure.in.in
===================================================================
RCS file: cdbakeoven/configure.in.in
diff -N cdbakeoven/configure.in.in
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ cdbakeoven/configure.in.in	31 Jan 2003 06:02:23 -0000
@@ -0,0 +1 @@
+DO_NOT_COMPILE="$DO_NOT_COMPILE cdbakeoven"
Index: kchat/configure.in.in
===================================================================
RCS file: kchat/configure.in.in
diff -N kchat/configure.in.in
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ kchat/configure.in.in	31 Jan 2003 06:02:23 -0000
@@ -0,0 +1 @@
+DO_NOT_COMPILE="$DO_NOT_COMPILE kchat"
Index: vimpart/src/Makefile.am
===================================================================
RCS file: /home/kde/kdeextragear-1/vimpart/src/Makefile.am,v
retrieving revision 1.10
diff -u -u -b -r1.10 Makefile.am
--- vimpart/src/Makefile.am	3 Oct 2002 23:21:09 -0000	1.10
+++ vimpart/src/Makefile.am	31 Jan 2003 06:02:23 -0000
@@ -8,14 +8,15 @@
 
 kde_module_LTLIBRARIES = libvimpart.la libxvim.la
 libxvim_la_SOURCES = xvim.cpp
+libxvim_la_LIBADD  = $(LIB_X11) $(LIB_POLL)
 libxvim_la_LDFLAGS = $(all_libraries) $(KDE_RPATH)
 
 xvim_SOURCES = xvimstart.cpp xvim_win.cpp 
 xvim_LDFLAGS = $(all_libraries) $(KDE_RPATH)
-xvim_LDADD = libxvim.la -L/usr/X11R6/lib -lX11 $(LIB_KDEUI)
+xvim_LDADD = libxvim.la -L/usr/X11R6/lib $(LIB_X11) $(LIB_KDEUI)
 
 libvimpart_la_SOURCES = kqeventutil.cpp vimembed.cpp kxeventutil.cpp vimdcop.cpp vimdocument.cpp vimview.cpp vimpart_factory.cpp vimwidget.cpp vimpartbrowserextension.cpp qkembed.cpp
-libvimpart_la_LIBADD   = -lktexteditor libxvim.la
+libvimpart_la_LIBADD   = -lktexteditor libxvim.la -lkdecore -lDCOP $(LIB_QT) $(LIB_KDEUI) -lkparts
 libvimpart_la_LDFLAGS = $(all_libraries) $(KDE_RPATH)
 
 # this is where the desktop file will go 
